app/views/registers/show.html.haml

```diff
@@ -109,8 +109,14 @@
                   %p
                     - if @search_term.present?
                       No results found for <strong>"#{@search_term}"</strong>
+                      %script
+                        var selectedInput = '';
```

Comment:
in this unused? I cannot find any other reference to it?

Reply:
It is unused - now fixed in subsequent push.